client/templates/adminMenu.html

```diff
@@ -25,23 +25,42 @@
                     </md-input-container>
                 </form>
             </md-menu-item>
-            <md-menu-divider></md-menu-divider>
-            <md-menu-item ng-repeat="speaker in speakers" class="md-indent admin-menu-item">
-                <md-button ng-click="removeSpeaker(speaker)" style="width: 0px" class="admin-menu-button md-no-focus">
-                    <i class="material-icons" style="margin-bottom: 15px; width: 0px">clear</i>
-                </md-button>
-                <span style="margin-top: 3px">{{speaker}}</span>
-            </md-menu-item>
-        </md-menu-content>
-    </md-menu>
-    <md-menu>
-        <button ng-click="logOut()" class="admin-menu-button">
-            LOG OUT
-        </button>
-        <md-menu-content>
-        </md-menu-content>
-    </md-menu>
-
-</md-menu-bar>
-
-
+        <md-menu-divider></md-menu-divider>
+        <md-menu-item ng-repeat="speaker in speakers" class="md-indent admin-menu-item">
+            <md-button ng-click="removeSpeaker(speaker)" style="width: 0px" class="admin-menu-button md-no-focus">
+                <i class="material-icons" style="margin-bottom: 15px; width: 0px">clear</i>
+            </md-button>
+            <span style="margin-top: 3px">{{speaker}}</span>
+        </md-menu-item>
+    </md-menu-content>
+</md-menu>
+<md-menu>
+    <button ng-click="$mdOpenMenu()" class="admin-menu-button">
+    RETWEETS
+    </button>
+    <md-menu-content>
+        <md-menu-item>
+            <md-button ng-click="setRetweetDisplayStatus('all')">
+                Show all
+            </md-button>
+        </md-menu-item>
+        <md-menu-item>
+            <md-button ng-click="setRetweetDisplayStatus('none')">
+                Show none
+            </md-button>
+        </md-menu-item>
+         <md-menu-item>
+            <md-button ng-click="setRetweetDisplayStatus('bristech_only')">
+                Show only @bristech retweets
```

Comment:
Fair enough; for what it's worth though, it should be possible to do so without actually checking that specifically, as all new tweets from any official user have the tag "official" in their corresponding "new_tweets" update. Again though, it seems fine for now.